additionalResolvers/flashcard.ts

```diff
@@ -0,0 +1,277 @@
+                    rewardPoints: ${assessment.metadata.rewardPoints},
+                    tagNames: [${assessment.metadata.tagNames.map(tag => `"${tag}"`)}]
+                },
+                assessmentMetadata: {
+                    skillPoints: ${assessment.assessmentMetadata.skillPoints},
+                    skillTypes: [${assessment.assessmentMetadata.skillTypes.map(skillType => `${skillType}`)}],
+                    initialLearningInterval: ${assessment.assessmentMetadata.initialLearningInterval}
+                },
+                items:[
+                    ${assessment.items.map(item => `{
+                        ${item.id != null ? `id:"${item.id}",` : ''}
+                        associatedSkills:[${item.associatedSkills.map(skill => `{ ${skill.id != null ? `id:"${skill.id}",` : ''} skillName:"${skill.skillName}"}`)}],
+                        associatedBloomLevels:[${item.associatedBloomLevels.map(level => `${level}`)}]
+                    }`)}
+                ]
+            }) {
+                id
+                items{
+                    id
+                    associatedSkills{
+                        id
+                        skillName
+                    }
+                    associatedBloomLevels
+                }
+            }
+            }`;
+            let updatedItems = await context.ContentService.Mutation.mutateContent({
+                root,
+                args:{contentId:_args.assessmentId},
+                selectionSet:selectionSet, 
+                context,
+                info});
+
+            let flashcardInput=_args.flashcardInput;
+            for (let item of updatedItems.updateAssessment.items) {
+                if (!oldItems.some(oldItem => oldItem.id === item.id)) {
+                    flashcardInput.itemId= item.id;
+                    break;
+                }
+            }
+            let selectionSetFlashcard=`{_internal_noauth_createFlashcard(input: {
+                itemId: "${flashcardInput.itemId}",
+                sides: [
+                    ${flashcardInput.sides.map(side=>`{
+                        label: "${side.label}",
+                        isQuestion: ${side.isQuestion},
+                        isAnswer: ${side.isAnswer},
+                        text: "${side.text}"}`)}
+                ]
+            }){
+                    itemId
+                    sides{
+                        label
+                        isQuestion
+                        isAnswer
+                        text
+                    }
+                }
+            }`;
+            let flashcard = await context.FlashcardService.Mutation.mutateFlashcardSet({
+                root,
+                 args: {
+                    assessmentId:_args.assessmentId
+                 },
+                 // we need to define a selection set manually here, otherwise it thinks we don't need any data
+                 // from this mutation and it won't actually be executed
```

Comment:
This code comment isn't really accurate, is it? You need the selection set to even mutate at all, as it contains the data you want to mutate